src/buildVariables.js

```diff
@@ -191,6 +191,13 @@ const buildUpdateVariables = (introspectionResults) => (
     resource,
     params
   );
+    let permitted_fields = null;
+    const resource_name = resource.type.name;
+    if (resource_name) {
+      permitted_fields = introspectionResults.types
+        .find((obj) => obj.name === `${resource_name}_set_input`)
+        ?.inputFields?.map((obj) => obj.name);
```

Comment:
The use of optional chaining requires an additional loader to be set up. 
@bharatkashyap - Can you configure this or rewrite the code a bit (with potentially more lines).